modules/accredited_representative_portal/db/migrate/20241223094500_make_poa_request_and_form_adjustments.rb

```diff
@@ -0,0 +1,24 @@
+class MakePoaRequestAndFormAdjustments < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def change
+    remove_index :ar_power_of_attorney_forms, name: 'idx_on_city_bidx_state_bidx_zipcode_bidx_a85b76f9bc'
+    remove_index :ar_power_of_attorney_forms, name: 'index_ar_power_of_attorney_forms_on_zipcode_bidx'
+    safety_assured { remove_columns :ar_power_of_attorney_forms, :city_bidx, :state_bidx, :zipcode_bidx }
+
+    add_column :ar_power_of_attorney_forms, :claimant_city_ciphertext, :string, null: false
+    add_column :ar_power_of_attorney_forms, :claimant_city_bidx, :string, null: false
+
+    add_column :ar_power_of_attorney_forms, :claimant_state_code_ciphertext, :string, null: false
+    add_column :ar_power_of_attorney_forms, :claimant_state_code_bidx, :string, null: false
+
+    add_column :ar_power_of_attorney_forms, :claimant_zip_code_ciphertext, :string, null: false
+    add_column :ar_power_of_attorney_forms, :claimant_zip_code_bidx, :string, null: false
+
+    add_index :ar_power_of_attorney_forms,
+              [:claimant_city_bidx, :claimant_state_code_bidx, :claimant_zip_code_bidx],
+              algorithm: :concurrently
+
+    add_column :ar_power_of_attorney_requests, :claimant_type, :string, null: false
```

Comment:
Is it absolutely necessary to add columns to both models at the same time? Generally I would avoid this unless I HAVE to

Reply:
@ojbucao can comment but I don't see why it's necessary. Is your suggestion be to split this up into two separate migrations, just so running & reverting seem cleaner?

Reply:
Actually, [we request](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-database-migrations#vets-apidatabasemigrations-Indexes) that indexes are modified in separate transactions. I can already here a few of you questioning this :) but we've seen things fail, and when it does, it requires a lot of work on our side. So please break this down into separate PRs for each of the index updates, and the two models (unless it's absolutely necessary the two models be updated at the same time)

Reply:
Different transactions or different PRs? 

All of these tables should have 0 records in all deployed environments as this isn't yet a production feature, or even one that's been tested in staging, does that make a difference to you?

Reply:
(I agree with the guidance on having index updates in separate transactions, just not sure of the case for making them separate PRs, they'll run separately anyway if split up into diff migrations no)

Reply:
I can split these up into separate migrations.

Reply:
guys - I didn't make the rules. Fighting me is only get your work less attention.

Reply:
I had intended to say _migrations_, which we generally would split into PRs, which is why I stated it that way. I suppose separate migrations in the same PR is ok,

Reply:
I wasn't trying to fight you on the rules! Correct me if I'm wrong and I will apologize profusely but you said two separate things and I was trying to distinguish